cmd/mb-main.go

```diff
@@ -33,6 +33,11 @@ var mbFlags = []cli.Flag{
 		Value: "us-east-1",
 		Usage: "specify bucket region; defaults to 'us-east-1'",
 	},
+	cli.StringFlag{
+		Name:  "panfs-path",
+		Value: "",
+		Usage: "specify path to bucket in panfs; defaults empty string means that global volume will be used as root for bucket",
```

Comment:
Typo default, not default**S**.

Reply:
Then I'll fix it to -> defaults **to**. Just to make help message consistent with the existing ones